tests/performance/log-results.js

```diff
@@ -20,42 +21,66 @@ const resultsFiles = [
 	},
 ];
 
-const performanceResults = resultsFiles.map( ( { file } ) =>
-	JSON.parse(
-		fs.readFileSync( path.join( __dirname, '/specs/' + file ), 'utf8' )
+// Results files recorded for the base commit.
+const baseResultsFiles = [
+	{
+		file: 'base-home-block-theme.test.results.json',
+		metricsPrefix: 'home-block-theme-',
+	},
+	{
+		file: 'base-home-classic-theme.test.results.json',
+		metricsPrefix: 'home-classic-theme-',
+	},
+];
+
+/**
+ * Parse test files into JSON objects.
+ *
+ * @param {Object[]} files
+ * @returns An array of parsed objects from each file.
+ */
+const parseResults = (files) => (
```

Comment:
Same here, missing spaces.

Reply:
Thanks Felix, Fixed in https://github.com/10up/wordpress-develop/pull/36/commits/583342746bfd0176d3d32e036f22d54d5eb09622